ocs_ci/deployment/aws.py

```diff
@@ -289,20 +289,25 @@ def deploy_prereq(self):
 
             # setup necessary env variables
             upi_env_vars = {
-                'INSTANCE_NAME_PREFIX': config.ENV_DATA['cluster_name'],
-                'AWS_REGION': config.ENV_DATA['region'],
-                'rhcos_ami': config.ENV_DATA['rhcos_ami'],
-                'route53_domain_name': config.ENV_DATA['base_domain'],
-                'vm_type_masters': config.ENV_DATA['master_instance_type'],
-                'vm_type_workers': config.ENV_DATA['worker_instance_type'],
-                'num_workers': str(config.ENV_DATA['worker_replicas'])
+                'INSTANCE_NAME_PREFIX': config.ENV_DATA.get('cluster_name'),
```

Comment:
If some of those variables are mandatory I would not use get function as than the job will continue and you will have None values set on the places you don't probably want if mandatory parameters. So if some of those parameters are mandatory for UPI I would keep old logic just accessing data by key as it will directly fail here at the start with clear error what is missing to be passed in the config.

Reply:
If they are truly mandatory the shell script is performing checks for them first thing so functionally it should be fine. However, the output from our code is easier to read than the output of the script so I'll revert some of these and only use `.get()` where necessary.